cocotb_build_libs.py

```diff
@@ -228,14 +212,17 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
     #
     #  libcocotb
     #
+    link_args = _extra_link_args(lib_name="libcocotb", rpath="$ORIGIN")
+    for lib_dir in python_lib_dirs:
+        link_args += _extra_link_args(rpath=lib_dir)
```

Comment:
Perhaps instead the `rpath` argument to `_extra_link_args` should be a list, making this:

```suggestion
    link_args = _extra_link_args(lib_name="libcocotb", rpath=["$ORIGIN"] + python_lib_dirs)
    for lib_dir in python_lib_dirs:
        link_args += _extra_link_args(rpath=lib_dir)
```

Reply:
Good idea. Changed `rpath` to `rpaths` list.